src/index.js

```diff
@@ -0,0 +1,70 @@
+function g(e) {
+  return document.getElementById(e);
+}
+function refresh(gameID) {
+  const baseUrl = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameID}/scores/`;
+  fetch(baseUrl)
```

Comment:
- [ ] As mentioned in the project requirements, kindly use the async and await functions with fetch to send and receive data. 😊